generators/client/templates/angular/src/main/webapp/app/core/auth/principal.service.ts.ejs

```diff
@@ -17,6 +17,9 @@
  limitations under the License.
 -%>
 import { Injectable } from '@angular/core';
+<%_ if (enableTranslation && !(authenticationType === 'oauth2')) { _%>
```

Comment:
This part is being refactored in another PR by William

Reply:
The second condition can be cleaner, do !== Instead and remove the extra braces around it